Color pickers added to options tab to allow users choose band colors

pacurtin opened the pull request.

peteharverson commented on public/prelert_swimlane_vis_controller.js:
```diff
@@ -263,7 +263,14 @@ module.controller('PrelertSwimlaneVisController', function ($scope, courier) {
       // plus an 'unknown' series for scores less than the 'low' threshold.
-      const colorBands = ['#e6e6e6', '#d2e9f7', '#8bc8fb', '#ffdd00', '#ff7e00', '#fe5050'];
+      //If color pickers have been used use new color choice else go to default colors.
+      const colorBands = [scope.vis.params.unknownThresholdColor ? scope.vis.params.unknownThresholdColor :'#e6e6e6',
```
Could you define the default colors in `prelert_swimlane_vis.js`, in the `params.defaults` block. That way the inputs are filled out with the default hexadecimal colors on opening. i.e.
```
params : {
      editor : require('plugins/prelert_swimlane_vis/prelert_swimlane_vis_params.html'),
      defaults : {
        interval : { display : 'Auto', val : 'auto' },
        lowThreshold: 0,
        warningThreshold: 3,
        minorThreshold: 25,
        majorThreshold: 50,
        criticalThreshold: 75,
        lowThresholdColor : '#d2e9f7',
        warningThresholdColor : '#8bc8fb',
        minorThresholdColor : '#ffdd00',
        majorThresholdColor : '#ff7e00',
        criticalThresholdColor : '#fe5050',
        tooltipNumberFormat: '0.0',
        showLegend: true
      },
      intervalOptions...
```

peteharverson commented on public/prelert_swimlane_vis_params.html:
```diff
@@ -1,53 +1,63 @@
       less-than="{{vis.params.warningThreshold}}"
       ng-required="true">
-    <kbn-info info="Defines lower threshold for values between {{vis.params.lowThreshold}} and {{vis.params.warningThreshold}}"></kbn-info>
+    <kbn-info info="Defines lower threshold for values between {{vis.params.lowThreshold}} and {{vis.params.warningThreshold}}."></kbn-info>  
```
No need for the full stop in the tooltip for the first field since it is only one sentence.

pacurtin commented:
Ok. I've refactored it to look at defaults for the colorpicker fields and changed that tooltip typo. It looks like it works but maybe give it a quick test before you merge it. Let me know if there is anything else you would like changed.
